### DISCUSSION TITLE
Welcome to TabHome Discussions!

### DISCUSSION THREAD

**Discussion by Yinbe** [Yinbe/TabHome#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Yinbe** [comment]
我们将讨论区用作与社区其他成员联系的地方。我们希望您：

-   提出你想知道的问题。
-   分享意见。
-   与其他社区成员互动。
-   欢迎他人，思想开放。请记住，这是我们的社区
-   一起建设💪。

首先，请在下方评论并介绍您对`TabHome`有什么想法，并告诉我们您希望在这个社区了解什么。

**Comment by lyl7410** [comment]
很实用的扩展！不知能否增加一个类似onetab的标签数据导入/导出和分享功能，这样更方便备份和分享了！

**Reply by Yinbe** [reply]
直接使用浏览器书签的导入导出功能就行了
